nengo_viz/static/viz_slider.js

```diff
@@ -393,6 +394,10 @@ VIZ.Slider.prototype.set_range = function() {
     }
 };
 
+VIZ.Slider.prototype.update_range = function(min, max) {
+    this.scale.domain([min, max]);
+}
+
 VIZ.Slider.prototype.layout_info = function () {
     var info = VIZ.Component.prototype.layout_info.call(this);
     info.min_value = this.scale.domain()[1];
```

Comment:
Yea, the prompt for reading in a new range is backwards, but I think it still works because the slider drag logic is backwards too. It should be an easy fix, I just didn't want to touch too much unrelated stuff in this branch